pkg/v2/resource/chart/resource.go

```diff
@@ -101,6 +105,36 @@ func (r *Resource) Name() string {
 	return Name
 }
 
+func (r *Resource) getGuestHelmClient(ctx context.Context, obj interface{}) (helmclient.Interface, error) {
```

Comment:
I was just wondering why the the HelmClient is not part of the `resource` struct, as `g8sclient` and `k8sclient`. Is this a differenct concept here?

Reply:
Ah, just saw its part of `guest`.

Reply:
Exactly, it is retrieved as the rest of guest clients according to this interface https://github.com/giantswarm/cluster-operator/blob/master/pkg/v2/guestcluster/spec.go#L11

Reply:
The helm client is configured for the guest cluster. Every guest cluster runs Tiller. We need a specific API endpoint configuration for the port forwarding so the client is unique and relates to the custom object.

Reply:
But still `g8sclient` and `k8sclient` are also retrieved via that interface and have an own field in `resource`, or am I mixing things up here?

Reply:
Ah didnt get Tim's comment in time. Thanks